flink-runtime/src/main/java/org/apache/flink/runtime/metrics/ReporterSetup.java

```diff
@@ -284,12 +304,19 @@ private static ReporterSetup createReporterSetup(
             try {
                 Optional<MetricReporter> metricReporterOptional =
                         loadReporter(reporterName, reporterConfig, reporterFactories);
+
+                Map<String, String> additionalVariables = reporterConfig.get(ADDITIONAL_VARIABLES);
```

Comment:
There's one issue here:

Currently, the keys of all variables are (unfortunately) formatted like `<job_id>`, but with the current documentation the brackets would be missing. We can _either_:
a) allow users to configure "my_tag:my_value" and add the brackets ourselves
b) mandate that users must configure "<my_tag>:my_value"

Considering that the variable exclusion does not use brackets (and that we want to get rid of these brackets eventually) we should go with a). So we'll have to build another map and adjust the keys accordingly.